provider/ansible.py

```diff
@@ -170,7 +170,13 @@ def distro_install(packages="ansible"):
     package_list = params.get_list("package_list", 'sshpass')
     try:
         path.find_command('ansible-playbook')
-    except path.CmdNotFoundError:
+        # If ansible debug plugin is required we need to enforce full
+        # ansible version installation instead of ansible-core
+        if params.get("ansible_callback_plugin") == "debug":
+            LOG_JOB.debug("ansible_callback_plugin is required")
```

Comment:
Suggest using the actual plugin name instead of `ansible_callback_plugin`

Reply:
Got it, it can avoid any possible confusion :-1: